tests/io/test_io.py

```diff
@@ -911,15 +910,15 @@ def test_open_datamet_datatree(datamet_file):
 
 
 @pytest.mark.parametrize("first_dim", ["time", "auto"])
-def test_cfradfial2_roundtrip(cfradial1_file, first_dim):
+def test_cfradfial2_roundtrip(cfradial1_file, make_temp_file, first_dim):
```

Comment:
Here we can use the first fixture, but we would need to create two filenames and then use the contextmanager like this:

```python
outfile = make_temp_file()
outfile1 = make_temp_file()

xradar.io.to_cfradial2(dtree0.copy(), outfile)
    # then open cfradial2 file
    with xr.open_datatree(outfile) as dtree1:
        # and write again
        fname2 = make_temp_file()
        xradar.io.to_cfradial2(dtree1.copy(), outfile1)
        # and open second cfradial2
        with xr.open_datatree(outfile1) as dtree2:
            # check equality
            for d0, d1, d2 in zip(dtree0.groups, dtree1.groups, dtree2.groups):
               ...
```

Reply:
OK